### PR TITLE
[icons] feat: new icons "ship", "cargo-ship"

### PR DESCRIPTION
**Pull request by jennyfan**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Adds icons `ship` and `ship-cargo`

#### Reviewers should focus on:

Make sure everything renders okay at appropriate sizes, and that the right files were changed.

#### Screenshot

Screen Shot 2021-12-16 at 6 39 09 PM
if any. -->

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @jennyfan! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/icons/resources/icons/icons.json [hunk 1]
```diff
@@ -3718,5 +3718,17 @@
...
+    {
+        "displayName": "Ship",
+        "iconName": "ship",
+        "tags": "transportation, ship, boat, cruise, yacht, move, supply, supply chain, trip, shipping",
+        "group": "interface"
+    },
+    {
+        "displayName": "Cargo Ship",
+        "iconName": "ship-cargo",
+        "tags": "transportation, cargo, cargo ship, freight, ship, boat, cruise, yacht, move, supply, supply chain, shipping container, shipping",
```
Comment (adidahiya) [inline packages/icons/resources/icons/icons.json]: don't need "cargo ship" when both individual words exist in the tags list already

```suggestion
        "tags": "transportation, cargo, freight, ship, boat, cruise, yacht, supply, supply chain, container, shipping",
```

File: packages/icons/resources/icons/icons.json [hunk 2]
```diff
@@ -3718,5 +3718,17 @@
         "iconName": "small-square",
         "tags": "square",
         "group": "miscellaneous"
+    },
+    {
+        "displayName": "Ship",
+        "iconName": "ship",
+        "tags": "transportation, ship, boat, cruise, yacht, move, supply, supply chain, trip, shipping",
```
Comment (adidahiya) [inline packages/icons/resources/icons/icons.json]:
```suggestion
        "tags": "transportation, ship, boat, cruise, yacht, supply, supply chain, trip, shipping",
```

File: packages/icons/resources/icons/icons.json [hunk 3]
```diff
@@ -3718,5 +3718,17 @@
...
+        "group": "interface"
+    },
+    {
+        "displayName": "Cargo Ship",
+        "iconName": "ship-cargo",
```
Comment (adidahiya) [inline packages/icons/resources/icons/icons.json]: let's just call this "cargo-ship" and make the API more intuitive... will need to rename the SVG files too

```suggestion
        "iconName": "cargo-ship",
```

**Review by adidahiya — CHANGES_REQUESTED** [review]

**Comment by adidahiya** [comment]
new icons look good in the [PR preview](https://50328-71939872-gh.circle-artifacts.com/0/packages/docs-app/dist/index.html#icons), though! 👍🏽

**Review by jennyfan — COMMENTED** [review]
Submitting reviewer changes

**Review by jennyfan — COMMENTED** [review]
Renaming files to cargo-ship

**Review by adidahiya — APPROVED** [review]

**Pull request merged**